### DISCUSSION TITLE
Is single nucleus RNA sequencing also compatible with scRNA-seq?

### DISCUSSION THREAD

**Discussion by dahun73** [satijalab/seurat#5647] [General]
Hello all!

As title mentioned, I'm curious about whether snRNA-seq is compatible with scRNA-seq using Seurat. 

Sequencing library is 10X chronium, and I used cell ranger.

Are there any tips or cautions for analyzing single nucleus(e.g. dimension reduction)?
I tried to find golden standard, but I could not find good reference.
It is also really helpful if you suggest any good paper :)

Thanks a lot!

**Comment by wudustan** [comment]
IIRC you expect a lower fraction of mitochondrial genes since you're targeting the nucleus not the whole cell, so adjust QC accordingly. Beyond that, you'll likely see a batch effect if you try to integrate a nuclear and a cellular seq from the same sample. But in terms of analysing the data by Seurat, you shouldn't have any issues as long as you have all the required cellranger outputs.

**Reply by dahun73** [reply]
@wudustan Thank you for reply! May I ask one more question? When I use 'cellranger count', the output always have this message.
![image](https://user-images.githubusercontent.com/52641420/155913970-4a36f385-bc17-4d8a-a40a-3b3d95d8e699.png)

I used '--include-intron' option. I read some documents for this message in 10X genomics website, and they say it is usual. Becaues there is no command for only single nuclues RNA sequencing data, this message came out(Usually, cellranger is focused for single cell). Am I right?

**Reply by wudustan** [reply]
It should be ok to proceed with analysis but I guess keep this in mind also when comparing your data with other data from single-cell and exon-enriched data

**Comment by oucstar** [comment]
@wudustan ,Hello, sorry to bother you, but I have seen your answer,I don't seem to understand the meaning of this sentence. Do you mean to add the parameter -- included-intron when comparing with the data of scrna-seq? I would appreciate it if you could answer me!

**Reply by wudustan** [reply]
`--include-intron` Is now a standard parameter used by cellranger as of version 7 release and is ran by default. If you have ran any data through cellranger 7 or higher this parameter has been already set. See: https://www.10xgenomics.com/support/software/cell-ranger/latest/release-notes/cr-release-notes#v7-0-0

From 10x technical documents:

> "In 10x Genomics Gene Expression data, intronic mapped reads account for 20-40% of the reads. These reads have traditionally not been counted by default in Cell Ranger. Recent data has indicated that intronic reads are usable data (they arise from polyA tracts in the transcripts, and are not generated via priming from genomic DNA)." 

[Source](https://kb.10xgenomics.com/hc/en-us/articles/4998628924429-Why-should-I-include-introns-for-my-single-cell-whole-transcriptome-Gene-Expression-data-analysis)

A much more in-depth guide can be found [here](https://assets.ctfassets.net/an68im79xiti/awNZTarmwqmwxKcvDv9wv/b05500661e36290b8ce59689ff889ea8/CG000376_TechNote_Antisense_Intronic_Reads_SingleCellGeneExpression_RevA.pdf)
